operator_ui/src/connectors/redux/reducers/jobs.js

```diff
@@ -38,6 +40,16 @@ export default (state = initialState, action = {}) => {
         items: Object.assign({}, state.items, action.data.specs)
       })
     }
+    case RECEIVE_DELETE_SUCCESS: {
+      return Object.assign({}, state, {
+        items: Object.assign(
+          {},
+          Object.keys(state.items)
+            .filter(i => i !== action.response)
+            .reduce((res, key) => ((res[key] = state.items[key]), res), {})
```

Comment:
How do you feel about using lodash [pick](https://lodash.com/docs/4.17.11#pick) here?